src/Sulu/Component/Content/Document/Subscriber/ShadowCopyPropertiesSubscriber.php

```diff
@@ -79,8 +81,11 @@ public function copyToShadows($document, NodeInterface $node)
         $categories = $this->getCategories($node, $document->getLocale());
         $navigationContext = $this->getNavigationContext($node, $document->getLocale());
 
-        foreach ($node->getProperties(self::SHADOW_BASE_PROPERTY) as $property) {
-            if ($property->getValue() === $document->getLocale()) {
+        foreach ($node->getProperties(self::SHADOW_BASE_PROPERTY) as $name => $property) {
+            $locale = $this->getLocale($name);
+            if ($node->getPropertyValueWithDefault(\sprintf(self::SHADOW_ON_PROPERTY, $locale), false)
+                && $property->getValue() === $document->getLocale()
+            ) {
```

Comment:
Doe we need to backport this to 2.4 / 2.5?

Reply:
done in https://github.com/sulu/sulu/pull/7310